diff --git a/dist/index.mjs b/dist/index.mjs
index 30bcabbd3a989e99499b26de68e1c1306ce7c2c9..f94f55e268bc849801909ffabab80d05153eaf3c 100644
--- a/dist/index.mjs
+++ b/dist/index.mjs
@@ -1345,7 +1345,6 @@ const pagesModule = defineNuxtModule({
       references.push({ types: useExperimentalTypedPages ? "vue-router/auto-routes" : "vue-router" });
       tsConfig.vueCompilerOptions ||= {};
       tsConfig.vueCompilerOptions.plugins ||= [];
-      tsConfig.vueCompilerOptions.plugins.push("vue-router/volar/sfc-route-blocks");
       if (useExperimentalTypedPages) {
         tsConfig.vueCompilerOptions.plugins.push({
           name: "vue-router/volar/sfc-typed-router",
